src/amo/reducers/addonsByAuthors.js

```diff
@@ -1,6 +1,8 @@
 /* @flow */
-import type { AddonType, ExternalAddonType } from 'core/types/addons';
+import invariant from 'invariant';
```

Comment:
oh cool 😍 when did you introduce it?

Reply:
Right?! Very recently, and it’s just piecemeal now but it’s very nice. A lot fewer silly error checking tests especially!